app/src/notifier/Notifier.js

```diff
@@ -47,7 +47,7 @@ class Notifier extends React.Component {
       <Snackbar
         anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
         message={message}
-        autoHideDuration={3000}
+        autoHideDuration={10000}
```

Comment:
is this milliseconds? meaning that the notifications will remain for 10 secs before auto-hiding?

if so, let's constant-ize this value so that its name can assist a future contributor in changing it and knowing exactly what they're changing (without having to go back to material-ui snackbar docs to look up the `autoHideDuration` prop

ex:
`const NOTIFY_AUTOHIDE_DURATION_MS = 10000;`

Reply:
yes